src/AppInstallerCommonCore/Filesystem.cpp

```diff
@@ -170,7 +170,7 @@ namespace AppInstaller::Filesystem
 
     bool VerifySymlink(const std::filesystem::path& symlink, const std::filesystem::path& target)
     {
-        const std::filesystem::path& symlinkTargetPath = std::filesystem::read_symlink(symlink).lexically_normal();
+        const std::filesystem::path& symlinkTargetPath = std::filesystem::read_symlink(symlink).weakly_canonical();

```

Comment:
This will probably fail the build, it should be std::filesystem::weakly_canonical(path)